source/testing/testing-controllers.md

```diff
@@ -33,43 +34,47 @@ export default Controller.extend({
 ```
 
 The `setProps` action directly sets one property, and calls the method to set the other.
-In our generated test, ember-cli already uses the `moduleFor` helper to set up a test
+In our generated test file, ember-cli already uses the `module` and the `setupTest` helpers to set up a test
 container:
 
 ```tests/unit/controllers/posts-test.js
-import { moduleFor, test } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
 
-moduleFor('controller:posts', {});
+module('Unit | Controller | posts', function(hooks) {
+  setupTest(hooks);
+});
 ```
 
-Next we use `this.subject()` to get an instance of the `PostsController` and
-write a test to check the action. `this.subject()` is a helper method from the
-`ember-qunit` library that returns a singleton instance of the module set up
-using `moduleFor`.
+Next we use the owner API to have access to the controller we'd like to test. Using the `this.owner.lookup` method we get an instance of the `PostsController` and can check the action in our test. The `this.owner.lookup` helper returns objects generated by the framework in your applications
```

Comment:
"have access"  -> "get access" or "gain access"

Reply:
> "we get an instance" 

we get **the** instance. controllers are singletons, there is only a single instance.